fixtures/location/update_location_fixtures.py

```diff
@@ -15,8 +15,8 @@
         "updateLocation": {
             "location": {
                 "name": "Nairobi",
-                "country": "Kenya",
-                "abbreviation": "KE"
+                "abbreviation": "KE",
+                "country": "CountryType.Kenya"
```

Comment:
Great work @C3real-kill3r.

Is there a reason why the country response field changed to this?
The original response would be more appropriate I believe.

Reply:
when you run the mutation query to update a location that's the output generated... reverting it to the previous format was causing the tests to fail....That was after updating the branch to the latest develop branch.